library/core/api/src/main/kotlin/de/lise/fluxflow/api/workflow/WorkflowElement.kt

```diff
@@ -0,0 +1,10 @@
+package de.lise.fluxflow.api.workflow
+
+/**
+ * Represents an element that can be part of a workflow.
+ */
+enum class WorkflowElement {
```

Comment:
Is there a reason not to use and extend the already existing `WorkflowObjectKind` enumeration?

Reply:
I didn't know that this enum already existed